pyecharts/utils.py

```diff
@@ -55,3 +55,50 @@ def json_dumps(data, indent=0):
     :param indent: 缩进量
     """
     return json.dumps(data, indent=indent, cls=UnknownTypeEncoder)
+
+
+def to_css_length(x):
+    """
+    Return the standard length string of css.
+    It's compatible with number values in old versions.
+    :param x:
+    :return:
+    """
+    if isinstance(x, (int, float)):
+        return '{}px'.format(x)
+    else:
+        return x
+
+
+def merge_js_dependencies(*chart_or_name_list):
+    """
+    Merge multiple dependencies to a total list.
+    This will ensure the order and unique in the items.
+    :param chart_or_name_list:
+    :return: A list containing dependency items.
+    """
+    front_must_items = ['echarts']  # items which must be included.
+    front_optional_items = ['echartsgl']
+    dependencies = []
+    fist_items = set()
+
+    def _add(_d):
+        if _d in front_must_items:
```

Comment:
'_d', I knew I have used it in pyechart. I am sorry. This will be picked up by pylint as too simple variable, no meaning and it will hurt future maintenance.

Reply:
个人观点 并不一定得全部按照 pylint 所要求的来 毕竟人是活的 工具是死的哈